### ISSUE TITLE
can't install fbs on xubuntu 16.04, python 3.5 using pyenv

### ISSUE THREAD

**Issue by potato20** [mherrmann/fbs#10]
getting this error

```
$ pip install fbs
Collecting fbs
  Using cached fbs-0.0.7.tar.gz
Installing collected packages: fbs
  Running setup.py install for fbs ... error
    Complete output from command /home/.../.pyenv/versions/3.5.2/envs/fbs/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-tza3c2lr/fbs/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-bost4s34-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/.../.pyenv/versions/3.5.2/envs/fbs/include/site/python3.5/fbs:
    running install
    running build
    running build_py
    creating build
    creating build/lib
    creating build/lib/fbs
    copying fbs/resources.py -> build/lib/fbs
    copying fbs/__main__.py -> build/lib/fbs
    copying fbs/platform.py -> build/lib/fbs
    copying fbs/builtin_commands.py -> build/lib/fbs
    copying fbs/__init__.py -> build/lib/fbs
    copying fbs/cmdline.py -> build/lib/fbs
    creating build/lib/fbs_runtime
    copying fbs_runtime/application_context.py -> build/lib/fbs_runtime
    copying fbs_runtime/system.py -> build/lib/fbs_runtime
    copying fbs_runtime/__init__.py -> build/lib/fbs_runtime
    copying fbs_runtime/signal_.py -> build/lib/fbs_runtime
    creating build/lib/fbs/freeze
    copying fbs/freeze/windows.py -> build/lib/fbs/freeze
    copying fbs/freeze/mac.py -> build/lib/fbs/freeze
    copying fbs/freeze/linux.py -> build/lib/fbs/freeze
    copying fbs/freeze/__init__.py -> build/lib/fbs/freeze
    copying fbs/freeze/arch.py -> build/lib/fbs/freeze
    creating build/lib/fbs/installer
    copying fbs/installer/__init__.py -> build/lib/fbs/installer
    creating build/lib/fbs/installer/windows
    copying fbs/installer/windows/__init__.py -> build/lib/fbs/installer/windows
    creating build/lib/fbs/installer/mac
    copying fbs/installer/mac/__init__.py -> build/lib/fbs/installer/mac
    creating build/lib/fbs/default_settings
    copying fbs/default_settings/mac.json -> build/lib/fbs/default_settings
    copying fbs/default_settings/base.json -> build/lib/fbs/default_settings
    copying fbs/installer/windows/Setup.nsi -> build/lib/fbs/installer/windows
    creating build/lib/fbs/installer/mac/yoursway-create-dmg
    copying fbs/installer/mac/yoursway-create-dmg/README.md -> build/lib/fbs/installer/mac/yoursway-create-dmg
    error: can't copy 'fbs/installer/mac/yoursway-create-dmg/support': doesn't exist or not a regular file
    
    ----------------------------------------
Command "/home/.../.pyenv/versions/3.5.2/envs/fbs/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-tza3c2lr/fbs/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-bost4s34-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/.../.pyenv/versions/3.5.2/envs/fbs/include/site/python3.5/fbs" failed with error code 1 in /tmp/pip-build-tza3c2lr/fbs/
```

**Comment by mherrmann** [comment]
I can reproduce this problem. Interestingly, it doesn't appear on Mac or Windows. I'll take a look and get back to you.

P.S. I redacted your user name from the output you posted.

**Comment by mherrmann** [comment]
Fixed in v0.0.8.

**Comment by potato20** [comment]
great, thx hermy
